### PR TITLE
Update LongConstantCompositeINTEL to LongCompositesINTEL capability after Headers change

### PR DESCRIPTION
**Pull request by vmaksimo**
Continue #2257
This fixes https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2261

The change is needed after https://github.com/KhronosGroup/SPIRV-Registry/commit/5d60c7d04589c3a84dee5abc6bc9bc5d1c045e3f and https://github.com/KhronosGroup/SPIRV-Headers/commit/d5acd42cb7cdaa5d0f2c5f739208eeeed395f491

### CODE REVIEW

File: include/LLVMSPIRVExtensions.inc [hunk 1]
```diff
@@ -43,7 +43,7 @@ EXT(SPV_INTEL_variable_length_array)
 EXT(SPV_INTEL_fp_fast_math_mode)
 EXT(SPV_INTEL_fpga_cluster_attributes)
 EXT(SPV_INTEL_loop_fuse)
-EXT(SPV_INTEL_long_constant_composite)
```
Comment (MrSidims) [inline include/LLVMSPIRVExtensions.inc:46]: the renaming is a breaking change
please first change SPIRVReader in release branches to accept both SPV_INTEL_long_constant_composite and SPV_INTEL_long_composites, then wait for a graceful period and only then proceed with the renaming in ToT

**Review by MrSidims — CHANGES_REQUESTED** [review]

File: include/LLVMSPIRVExtensions.inc (hunk 1 above)
Comment (vmaksimo) [reply include/LLVMSPIRVExtensions.inc:46]: Gotcha, thanks!

**Comment by vmaksimo** [comment]
Please take a look @asudarsa @LU-JOHN @bwlodarcz @VyacheslavLevytskyy

**Comment by VyacheslavLevytskyy** [comment]
LGTM

**Review by LU-JOHN — APPROVED** [review]
LGTM

**Comment by LW-archlinux** [comment]
This solves https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2261 when built against https://github.com/KhronosGroup/SPIRV-Tools/commit/e03c8f5c8ed77f0b1a2408d55bb8f00a7ee9793c and https://github.com/KhronosGroup/SPIRV-Headers/commit/1c6bb2743599e6eb6f37b2969acc0aef812e32e3

Please add `closes: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2261 ` or similar to the commit message.

**Comment by asudarsa** [comment]
Can we please add a TODO in include/LLVMSPIRVExtensions.inc for the required change to happen in the future?

Thanks

**Review by asudarsa — APPROVED** [review]
LGTM. Thanks

**Pull request merged**